src/python/director/cameraview.py

```diff
@@ -717,7 +716,6 @@ def update(self, robotModel):
 
 
 def addCameraView(channel, viewName=None, cameraName=None, imageType=-1):
-
     cameraName = cameraName or channel
     imageManager.queue.addCameraStream(channel, cameraName, imageType)
```

Comment:
addCameraStream() is called here.  The only special case is to handle the "CAMERA" channel, where we need to also call addCameraStream("CAMERA", "CAMERA_LEFT", lcmbotcore.images_t.LEFT)